connector/src/test/java/it/eng/connector/util/TestUtil.java

```diff
@@ -0,0 +1,10 @@
+package it.eng.connector.util;
+
+public class TestUtil {
+	//all values in this class are from the initial_data.json
+	public static final String USER_DETAILS = "connector@mail.com";
```

Comment:
Can you also rename this one to CONNECTOR_USER?

Reply:
No problemo 👍